pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -86,6 +106,6 @@ func (c *ClientCache) ClientForVersion(version string) (*client.Client, error) {
 		return nil, err
 	}
 
-	c.clients[config.Version] = client
+	c.clients[config.GroupVersion.String()] = client
```

Comment:
the only thing I dislike about config.GroupVersion being a pointer is trying to figure out if it is possible for it to be nil in places like this